test/optim/test_optim.py

```diff
@@ -813,6 +813,10 @@ def _test_foreach_memory(self, optimizer_pairs_with_flags):
                 # with capturable in Adam(W), we have 2 extra intermediates for the bias_corrections
                 # with Adadelta, we have 2 extra for (acc_delta + eps) and (square_avg + eps)
                 nintermediates = 3
+                if optimizer_constructor.__name__ == "NAdam":
+                    # with capturable in NAdam, we have 3 extra intermediates for the
+                    # bias_correction, mus, and mu_nexts
+                    nintermediates = 5
```

Comment:
Hmmm, the above comment has 2 extra leading to 3. And this one has 3 extra leading to 5?